corehq/apps/sms/tasks.py

```diff
@@ -154,12 +181,21 @@ def handle_outgoing(msg):
             # Requeue the message and try it again shortly
             return True
 
+    if max_simultaneous_connections:
```

Comment:
Thoughts on explicitly testing for `is not None` here (and below), or else noting in `get_max_simultaneous_connections` that a return value of zero will also be ignored?

Reply:
@orangejenny i don't think we'd be able to support a value of 0 since that would mean that we'd never have any connections open